README.md

```diff
@@ -1,23 +1,36 @@
-A Python library for maintaining and managing DANDI metadata schemata. The
-library helps create and validate DANDI schema-compliant metadata for Dandisets
-and assets.
+# DANDI Schema
 
-To use: `pip install dandischema`
+The dandi-schema repository is a Python library for maintaining and managing DANDI metadata schemata. 
 
-Every Dandiset and associated asset has a metadata object that can be retrieved using
-the DANDI API.
+## Installation
 
-This library uses [Pydantic](https://github.com/samuelcolvin/pydantic) to implement
+`pip install dandischema`
+
+## Description
+
+Every `Dandiset` and associated asset has a metadata object that can be retrieved using
+the DANDI API. This library helps create and validate DANDI schema-compliant metadata for `Dandisets`
```

Comment:
```suggestion
the DANDI API. This library helps create and validate DANDI schema-compliant metadata for Dandisets
```

Reply:
Then I agree that, for consistency, it should be formatted as a code element.

But on a broader level, I don't think that is the right choice. Can you point me to the style policy/discussion where this decision was made?

Reply:
Let's make the decision now to remove the code element from the term "Dandiset". Can you update the stylesheet, @melster1010?

Reply:
@waxlamp sure - I'll have to update the stylesheet, remove the special formatting from "Dandiset" in the Handbook files, as well as this readme (as separate PR)